docs/kratos/social-signin/95_account-linking.mdx

```diff
@@ -67,5 +93,13 @@ Automatic account linking can be a security risk. Consider this scenario:
 1. Your system, detecting duplicate accounts, prompts for account linking.
 1. Malicious actors link the accounts, gaining access to John's account.
 
-To prevent this users need to verify an additional credential before the accounts can be linked. In the scenario above, the
+To prevent this, users need to verify an additional credential before the accounts can be linked. In the scenario above, the
 malicious actors would be prompted to enter the password associated with the `jon@doe.com` identifier.
+
+### Hide irrelevant authentication methods
```

Comment:
Which methods do we hide specifically, or what makes them irrelevant? 
hard to tell from the headline what this means